### PR TITLE
Added Word shuffle game

### LINKED ISSUE #4406: [New game]:  Word Shuffle Game Request
### 🎮 Game Request

The Word Shuffle game challenges players to unscramble a set of letters to form meaningful words within a given time frame. Players are presented with a grid of letters, and they must rearrange these letters to create valid words. The game provides hints and shuffle options to assist players in finding words. Each level becomes progressively more difficult, with longer words and shorter time limits.

### Point down the features

Features:

Word Unscrambling: Players unscramble letters to form valid words.
Time Limit: Each level has a time limit within which players must complete the word unscrambling task.
Hints: Players can use hints to reveal letters or words to aid them in solving the puzzle.
Shuffle Option: Players can shuffle the letters to get a different arrangement, potentially revealing new word combinations.
Progressive Difficulty: As players advance through levels, the complexity of the word puzzles increases.
Score Tracking: The game keeps track of players' scores based on the number of words solved and the time taken.
Level System: Players progress through different levels, each with its own unique set of word puzzles.
Responsive Design: The game is designed to be playable on various devices with different screen sizes.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by thevijayshankersharma**
## PR Description 📜

Fixes #4406 

<hr>
 
## Mark the task you have completed ✅

- [ ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

image

<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @thevijayshankersharma ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -328,12 +328,11 @@ This repository also provides one such platforms where contributers come over an
 |[Color The Page](https://github.com/kunjgit/GameZone/tree/main/Games/Color_The_Page)|
 |[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game)   |
 | [Snake](https://github.com/kunjgit/GameZone/tree/main/Games/snake)                               |
-
 |[Chess_Game_computer](https://github.com/kunjgit/GameZone/tree/main/Games/Chess_Game_computer)   |
-
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light)   |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
-| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) |
+|[Dsa_quiz_game](https://github.com/kunjgit/GameZone/tree/main/Games/Dsa_quiz_game)   |
```
Comment (ishita-43) [inline README.md:334]: change with rapid click frenzy one

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by thevijayshankersharma** [comment]
@ishita-43 Is it okay now ??

File: README.md [hunk 2]
```diff
@@ -328,16 +328,18 @@ This repository also provides one such platforms where contributers come over an
...
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light)   |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
-| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) |
+|[Dsa_quiz_game](https://github.com/kunjgit/GameZone/tree/main/Games/Dsa_quiz_game)   |   
+| [Word_Shuffle_Game](https://github.com/example/word_shuffle_game) |
 | [Dsa_quiz_game](https://github.com/kunjgit/GameZone/tree/main/Games/Dsa_quiz_game) |
 | [Gravity_Simulation_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Gravity_Simulation_Game) |
 | [Anagarm-Word-Game](https://github.com/kunjgit/GameZone/tree/main/Games/Anagarm-Word-Game) |
-| [Alphabet-and-Vowels](https://github.com/kunjgit/GameZone/tree/main/Games/Alphabet-and-Vowels) |
+| [Brick Buster Game](https://github.com/kunjgit/GameZone/tree/main/Games/Brick Buster) |
```
Comment (ishita-43) [inline README.md:339]: remove this

File: README.md [hunk 3]
```diff
@@ -328,16 +328,18 @@ This repository also provides one such platforms where contributers come over an
...
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) |
```
Comment (ishita-43) [inline README.md:340]: remove this

File: README.md [hunk 4]
```diff
@@ -328,16 +328,18 @@ This repository also provides one such platforms where contributers come over an
...
+|[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
```
Comment (ishita-43) [inline README.md:341]: remove this

File: README.md [hunk 5]
```diff
@@ -328,16 +328,18 @@ This repository also provides one such platforms where contributers come over an
...
+| [Intellect Quest](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Intellect_Quest)|
```
Comment (ishita-43) [inline README.md:342]: remove this duplicate entry

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Ensure no other duplicate entries are made by you

**Comment by thevijayshankersharma** [comment]
@ishita-43 @kunjgit All things is correct now. You can merge it.

**Review by ishita-43 — APPROVED** [review]
Good to go!

**Comment by kunjgit** [comment]
Thank you  @thevijayshankersharma , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**